README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
"pick random x to y" block behaves kinda strange if x >= y. I'm not sure what the behavior should be, but I think it should be consistent (i.e. either include the endpoints or don't)

| x | y | output |
| - | - | - |
| 4 | 4 | [4] |
| 4 | 3 | [4] |
| 4 | 2 | [3] |
| 4 | 1 | [2, 3] |
| 4 | 0 | [1, 3] |

Reply:
I'll make it an error